Sources/SwiftDocCTestUtilities/SymbolGraphCreation.swift

```diff
@@ -1,23 +1,19 @@
 /*
  This source file is part of the Swift.org open source project
 
- Copyright (c) 2021-2024 Apple Inc. and the Swift project authors
+ Copyright (c) 2021-2025 Apple Inc. and the Swift project authors
  Licensed under Apache License v2.0 with Runtime Library Exception
 
  See https://swift.org/LICENSE.txt for license information
  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-
-import Foundation
-public import XCTest
+package import Foundation
 package import SymbolKit
 package import SwiftDocC
 
 // MARK: - Symbol Graph objects
 
-extension XCTestCase {
-    
     package func makeSymbolGraph(
```

Comment:
I know it'll leave a larger diff, but should we un-indent all this code that used to be in the extension?